aptos-move/aptos-vm-types/src/module_and_script_storage/state_view_adapter.rs

```diff
@@ -31,7 +31,10 @@ use move_vm_runtime::{
 };
 use move_vm_types::{
     code::{ModuleBytesStorage, ModuleCode},
-    loaded_data::runtime_types::{StructType, Type},
+    loaded_data::{
+        runtime_types::{StructType, Type},
+        struct_name_indexing::StructNameIndex,
```

Comment:
stupid question, what causes all these files to just have added StructNameIndex in uses but no other change?

Reply:
Because of Delegate, impl is auto generated but the imports are still needed

Reply:
makes sense!